src/components/NavMenu/NavMenu.tsx

```diff
@@ -25,17 +25,14 @@ import {
 } from '@floating-ui/react-dom-interactions';
 import { styled } from '@storybook/theming';
 import mergeRefs from 'react-merge-refs';
-import { color, shadows } from '../shared/styles';
+import { MenuPanel } from '../Menu/Menu';
 import { NavMenuButton } from './NavMenuButton';
 
-export const NavMenuPanel = styled.div`
+export const NavMenuPanel = styled(MenuPanel)`
   padding: 12px;
   width: 288px;
 
-  background: ${color.lightest};
   border-radius: 5px;
```

Comment:
Is there a reason this is different from `MenuPanel`'s `border-radius`? If not, which is correct?

Reply:
Initially the `NavMenuPanel` and `MenuPanel` were more different. That's why I didn't use the same component. I think now the difference is width and border radius. `NavMenuPanel` should be `5px`